__tests__/posts.ts

```diff
@@ -1068,6 +1068,51 @@ describe('query post', () => {
       'FORBIDDEN',
     );
   });
+
+  describe('clickbaitTitleDetected', () => {
+    const LOCAL_QUERY = /* GraphQL */ `
+      query Post($id: ID!) {
+        post(id: $id) {
+          clickbaitTitleDetected
+        }
+      }
+    `;
+
+    it('should return true if clickbait title probability is above threshold', async () => {
```

Comment:
Can we maybe add a test where it's more mimicked on floats?
Just to ensure float comparison never breaks down the line?

Reply:
also since we use parseFload, can we also have test for string being there instead of number in JSON, just to be sure it also works

Reply:
Changed it to be `'1.99'`

Reply:
it could be two different tests, default which will most probably be float already, and separate one to handle the string case, better to detect what breaks

Reply:
Added separate test for float